### DISCUSSION TITLE
How to apply a change in the integration's configuration settings?

### DISCUSSION THREAD

**Discussion by erikgeurts** [Hyundai-Kia-Connect/kia_uvo#713] [Q&A] [answered]
When I change something in the settings of the integration, for example when I change the scan interval, and click Submit, will this take effect automatically, or do I need to do something else, like reloading the integration or restarting Home Assistant?

**Comment by fuatakgun** [comment, marked as answer]
go to devices & services, click on Configure and change the settings, that is it. It will reload it for you automatically
